src/test/internals.test.ts

```diff
@@ -0,0 +1,43 @@
+import { afterEach, beforeAll, beforeEach, describe, expect, it } from "vitest";
+import Phaser from "phaser";
+import GameManager from "#app/test/utils/gameManager";
+import { Species } from "#app/enums/species.js";
+import { Abilities } from "#app/enums/abilities.js";
+
+describe("Internals", () => {
+  let phaserGame: Phaser.Game;
+  let game: GameManager;
+
+  beforeAll(() => {
+    phaserGame = new Phaser.Game({
+      type: Phaser.HEADLESS,
+    });
+  });
+
+  afterEach(() => {
+    game.phaseInterceptor.restoreOg();
+  });
+
+  beforeEach(() => {
+    game = new GameManager(phaserGame);
+  });
+
+  it("PokemonSpeciesForm - .getAbilityCount() and .getAbility()", async () => {
```

Comment:
Test names should be a "narrative". The "it" is part of it.

e.g. "it should provide Eevee with 3 defined abilities"
=> 
```suggestion
  it("it should provide Eevee with 3 defined abilities", async () => {
```